modules/ROOT/pages/introduction/intro-config-use-fd.adoc

```diff
@@ -0,0 +1,165 @@
+image::introduction/intro-config-fd-scheduler-fixed.png[Configuration for a fixed scheduling strategy]
++
+
+The following configuration uses a cron expression to initiate an action every second:
++
+.Select `Cron` in the *Scheduling Strategy* field to use a cron expression to specify when to initiate a flow.
+image::introduction/intro-config-fd-scheduler-cron.png[Configuration using a cron expression]
+
+For more information about configuring `Scheduler`, see xref:mule-runtime::scheduler-concept.adoc[Scheduler Endpoint]
+
+
+[[connector-specific-trigger]]
+=== Use a Connector-Specific Trigger
+
+If a connector has associated triggers, you can use a connector-specific trigger to initiate the flow. For example, if you are configuring an app that contains an Amazon SQS Connector operation, you can use the `Amazon SQS Receive Messages` trigger to initiate a flow each time the configured Amazon SQS queue receives a new message.
+
+To start the configuration for a connector-specific trigger, follow these steps:
+
+. Follow the procedure described in <<configure-trigger,Configure the Trigger For the Flow>>.
+. In the *Search* field, search for and select the connector.
+. If the connector has multiple triggers, select the trigger to use. (If the connector has one trigger, that trigger is selected automatically.)
+
+Each connector-specific trigger has a unique configuration, but most configurations require the following actions:
+
+* Completing the required fields.
+* Selecting an existing connection or configuring a new one.
+
+For more information about configuring a connector-specific trigger, see the reference guide for the associated connector.
+
+== Configure the Target Component
+
+After you configure a trigger for a connector, you can configure the connector as the target component:
+
+NOTE: Depending on the connector you are working with, the step order in the following procedure might vary.
+
+. Click the *+* icon next to the trigger card.
++
+.Clicking the *+* icon brings up a screen where you can select the target connector operation.
+image::introduction/intro-config-fd-plus-target.png[Click the Plus Sign]
++
+. In the *Select a component* screen, search for and select the target connector.
+. Select the target operation.
+. Select an existing connection configuration or add a new one.
+. To add a new connection:
+.. Click *Add Connection*.
+.. Specify a connection name.
+.. In the *Connection Type* field, select an authentication type.
+.. Complete the required fields for the selected authentication type.
+. Click *Test* to confirm that Mule can connect with the specified server using the configured authentication.
+. Click *Save*.
+. Complete the required fields, if any.
+. Optionally click *Advanced* to create a reconnection policy or expiration policy.
++
+NOTE: The available options vary by connector.
++
+. If you made additional configuration changes, click *Save* again.
```

Comment:
Do we need to be specific about what these changes are in addition to (after the first time you clicked Save)?